src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/assets_transfers/per_recipient/per_recipient_non_fungibles_transfer.rs

```diff
@@ -0,0 +1,109 @@
+use crate::prelude::*;
+
+impl PerRecipientNonFungiblesTransfer {
```

Comment:
The `PerRecipientNonFungiblesTransfer ` type is declared with macro in `transfer_types.rs`